go/README.md

```diff
@@ -25,25 +25,31 @@ go build
 APP_PORT=8000 \
 PLAID_CLIENT_ID=[CLIENT_ID] \
 PLAID_SECRET=[SECRET] \
-PLAID_PUBLIC_KEY=[PUBLIC_KEY] \
 PLAID_PRODUCTS=[PRODUCTS] \
 PLAID_COUNTRY_CODES='US' \
 go run server.go
 
 # Go to http://localhost:8000
 ```
+### Link token creation and server-side configuration.
+The new way to initialize Plaid Link is to pass Plaid Link initialization
```

Comment:
Is "new" needed here? I would assume most people using the Quickstart are new to using Plaid anyway, so they wouldn't know about the public_key way, and this may be confusing?

Reply:
➕ Feels like a good opportunity to clarify that link tokens is the recommended / blessed way now.